cpg-concepts/build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, Fraunhofer AISEC. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+plugins {
+    id("cpg.frontend-conventions")
+}
+
+publishing {
+    publications {
+        named<MavenPublication>("cpg-concepts") {
+            pom {
+                artifactId = "cpg-concepts"
+                name.set("Code Property Graph - Concepts")
+                description.set("A 'concepts' extension for the CPG")
+            }
+        }
+    }
+}
+
+dependencies {
+    // to evaluate some test cases
+    testImplementation(project(":cpg-analysis"))
+
+    // to evaluate some test cases
+    testImplementation(project(":cpg-language-python"))
```

Comment:
We can't do that. We need to move tests that depend on languages into the integration test. But we can't depend on the `cpg-language-python` frontend if it's not there. So we have to somewhere state that you need to enable the python frontend for the integration tests, but cannot make an explicit dependency on it :(

Reply:
moved to integrationTestImplementation